rosetta-runtime/src/main/java/com/rosetta/model/metafields/MetaFields.java

```diff
@@ -42,7 +38,9 @@ public interface MetaFields extends RosettaModelObject, GlobalKeyFields, MetaDat
 	/*********************** Getter Methods  ***********************/
 	String getScheme();
 	String getTemplate();
+	@Deprecated
 	String getLocation();
+	@Deprecated
```

Comment:
I thought it would be `getLocation()` and `getKey()` that would be deprecated, in favour of `getScopedKey()`? I don't think `address` should be deprecated.